debs/bionic/archivematica/debian-MCPClient/control

```diff
@@ -0,0 +1,49 @@
+Source: archivematica-mcp-client
+Section: utils
+Priority: extra
+Maintainer: Santiago Collazo <sysadmin@artefactual.com>
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.9.8
+Homepage: http://archivematica.org
+
+Package: archivematica-mcp-client
+Architecture: i386 amd64
+Depends:
+	${shlibs:Depends},
+	${misc:Depends},
+	archivematica-common (>=1.6.0),
+	atool,
+	bagit,
+	bulk-extractor,
+	clamav,
+	clamav-daemon,
+	ffmpeg (>= 7:3.0),
+	libavcodec-extra-56|libavcodec-extra57,
+	fits (>= 0.8.4),
+	gearman,
+	imagemagick (>=8:6.7.7.10),
+	inkscape,
+	jhove,
+	libimage-exiftool-perl,
+	libxml2-utils,
+	logapp,
+	md5deep,
+	mediaconch (>= 16.12),
+	mediainfo,
+	nailgun,
+	nailgun-client,
+	nfs-common,
+	openjdk-8-jre-headless,
+	p7zip-full,
+	pbzip2,
+	postfix,
+	readpst,
```

Comment:
The `readpst` package doesn't exist on Bionic: https://launchpad.net/ubuntu/bionic/amd64/readpst . The readpst command is included in pst-utils:

```
root@am18bionic:~# which readpst 
/usr/bin/readpst
root@am18bionic:~# dpkg -S /usr/bin/readpst
pst-utils: /usr/bin/readpst
```

So we can include the `pst-utils` dependency instead of `readpst`.